xayn-ai/src/coi/system.rs

```diff
@@ -186,17 +186,11 @@ impl CoiSystem {
         })
     }
 
-    /// Computes the relevance/weights of the cois.
+    /// Computes the relevances of the cois.
     ///
-    /// The weights are computed from the view counts and view times of each coi and they are not
-    /// normalized. The horizon specifies the time since the last view after which a coi becomes
-    /// irrelevant.
-    #[allow(dead_code)]
-    fn compute_weights<CP: CoiPoint + CoiPointStats>(
-        &self,
-        cois: &[CP],
-        horizon: Duration,
-    ) -> Vec<f32> {
+    /// The relevance of each coi is computed from its view count and view time relative to the
+    /// other cois. It's an unnormalized score from the interval `[0, ∞)`.
+    fn compute_relevances<CP: CoiPoint + CoiPointStats>(&self, cois: &[CP]) -> Vec<f32> {
```

Comment:
Also here I think is better if we have a core that does not depends on self and the use it with the configuration we have.

Reply:
i guess the same argument applies as in the other pr: as the whole module is structured this way, this method is as well. as we agreed to change it in another pr, i'd like to leave it as is for now.